manifest.xml

```diff
@@ -103,14 +101,14 @@
           </ExtensionPoint>
         </DesktopFormFactor>
       </Host>
+    <Hosts>
```

Comment:
I don't think this should be here.  It creates an incomplete balance of tags (there is a missing </Hosts> tag) that doesn't validate or deploy when running 'npm start'.  It wasn't needed before, and I don't see why it's needed now.

Reply:
Good catch. Fixed.